README.md

```diff
@@ -133,10 +135,14 @@ High level pseudo-code:
         - Figure out Huma naming & type
         - Convert validation to Huma's JSON-Schema tags
       - Generate `FromProto` and `ToProto` converter methods
-  - Write out `$BASENAME.huma.go`
+  - Write out `$DIRNAMEhuma/$BASENAME.huma.go`
```

Comment:
nit: should there be a `/` between DIRNAME and huma?

Reply:
Nope, that's on purpose! For example, `package1/file1.proto` becomes `package1huma/file1.huma.go` while the official Go protobuf plugin produces `package1/file1.pb.go`. It's because we reuse the message names for Go structs and so need a different Go package name, which in turn requires a different directory name. I chose appending over additional nesting for simplicity.